core/templates/dev/head/pages/footer.html

```diff
@@ -22,7 +22,7 @@ <h4 translate="I18N_FOOTER_TEACH_LEARN_ALL_CAPS"></h4>
       <div class="col-sm-3">
         <h4 translate="I18N_FOOTER_CONTRIBUTE_ALL_CAPS"></h4>
         <ul>
-          <li><a href="http://oppiafoundation.org/get-involved/" translate="I18N_FOOTER_GET_INVOLVED"></a></li>
+          <li><a href="http://oppiafoundation.org/volunteer" translate="I18N_FOOTER_GET_INVOLVED"></a></li>
```

Comment:
Thanks! Can you change to https as well?

Reply:
Done!